hat/assets/js/apps/Iaso/domains/entities/components/Filters.tsx

```diff
@@ -196,15 +179,6 @@ const Filters: FunctionComponent<Props> = ({ params, isFetching }) => {
         [filters],
     );
 
-    const groupsInputComponent = useMemo(() => {
-        return createGroupsInputComponent({
-            isFetchingGroups,
-            handleChange,
-            filters,
-            groups,
-        });
-    }, [filters, groups, handleChange, isFetchingGroups]);
-
```

Comment:
This was a hidden component. I took it out. It introduces some repetition, but is more legible I think, and avoids the unnecessary `useMemo`